phil/s013-course-periods.md

```diff
@@ -13,7 +44,7 @@ Screens:
 ```coffee
```

Comment:
Well this approach totally works until next January, so I'm in favor of sticking with the dumb approach.